src/sql-parser/tests/sqlparser_common.rs

```diff
@@ -153,6 +153,15 @@ fn datadriven() {
         let input = tc.input.trim();
         match parser::parse_expr(input) {
             Ok(s) => {
+                // TODO(def-) Can we enable this? Fails with LIKE => ~~ currently
```

Comment:
With LIKE or with ILIKE, see https://github.com/MaterializeInc/materialize/issues/18494

Reply:
Both I think. Yes, that issue is covering one part of the problem and can be closed when this is fixed.